fastlane/helper/semantic_release_helper.rb

```diff
@@ -0,0 +1,97 @@
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Helper
+    class SemanticReleaseHelper
+      # class methods that you define here become available in your action
+      # as `Helper::SemanticReleaseHelper.your_method`
+      #
+      def self.git_log(pretty, start)
+        command = "git log --pretty='#{pretty}' --reverse #{start}..HEAD"
+        Actions.sh(command, log: false).chomp
+      end
+
+      def self.parse_commit(params)
+        commit_subject = params[:commit_subject].strip
+        commit_body = params[:commit_body]
+        releases = params[:releases]
+        codepush_friendly = params[:codepush_friendly]
+        pattern = /^(docs|fix|feat|chore|style|refactor|perf|test)(\((.*)\))?(!?)\: (.*)/
+        breaking_change_pattern = /BREAKING CHANGES?: (.*)/
+        codepush_pattern = /codepush?: (.*)/
```

Comment:
As above, probably want a `\s*` rather than a hardcoded single space between colon and capture